docs/troubleshooting-guide/linked-filters.md

```diff
@@ -0,0 +1,37 @@
+# My linked filters don't work
+
+Metabase lets you [link filters][linked-filter-gloss] so that (for example) if a dashboard contains both a "State" and a "City" filter, the "City" filter only shows cities in the state selected by the "State" filter; the article ["Linking filters in dashboards"][learn-linking] explains how to set them up.
+
+In order to fix problems associated with linked filters, you need a clear understanding of how they work:
+
+1. A filter isn't part of a specific question. Instead, a filter is added to a dashboard and its value is used to fill in a variable in a question (or in several questions).
+
+2. In order for Metabase to display a dropdown list of possible filter values, it must know that the filter corresponds to a category. If the question is created using the Notebook Editor, this happens automatically, since Metabase analyzes the query as it turns the graphical representation into SQL.
```

Comment:
Lets avoid writing about SQL here.
```suggestion
2. In order for Metabase to display a dropdown list of possible filter values, it must know that the column corresponds to a category. If the question is created from tables via the graphical editor (Simple question or Custom question), this happens automatically, since Metabase has knowledge about the table and columns from synchronization.
```